src/main/java/hudson/plugins/performance/PerformancePublisher.java

```diff
@@ -147,7 +180,12 @@ public PerformancePublisher(int errorFailedThreshold,
                             boolean modeOfThreshold,
                             boolean compareBuildPrevious,
                             List<? extends PerformanceReportParser> parsers,
-                            boolean modeThroughput) {
+                            boolean modeThroughput,
+                            boolean modeEvaluation,
+                            List<? extends AbstractConstraint> constraints,
+                            boolean ignoreFailedBuilds,
+                            boolean ignoreUnstableBuilds,
+                            boolean persistConstraintLog) {
```

Comment:
Tried to overload the constructor - This lead to the following error (while starting Jenkins):

[ERROR] Failed to execute goal org.apache.maven.plugins:maven-compiler-plugin:2.3.2:compile (default-compile) on project performance: Compilation failure
[ERROR] error: javax.annotation.processing.FilerException: Attempt to reopen a file for path C:\performance-plugin\target\classes\hudson\plugins\performance\PerformancePublisher.stapler

Content of PerformancePublisher.stapler:
#Fri Feb 19 14:22:40 CET 2016
constructor=errorFailedThreshold,errorUnstableThreshold,errorUnstableResponseTimeThreshold,relativeFailedThresholdPositive,relativeFailedThresholdNegative,relativeUnstableThresholdPositive,relativeUnstableThresholdNegative,nthBuildNumber,modePerformancePerTestCase,comparisonType,modeOfThreshold,compareBuildPrevious,parsers,modeThroughput

Any idea about this problem?